### PR TITLE
MP-7294 [BS-8103] NOX - Define and implement custom type "Formula" #82

### PR DESCRIPTION
**Pull request by emir-sehovic1**

### CODE REVIEW

**Review by gjorgji-grgovski-authoritypartners — APPROVED** [review]
All good with just a small comment, please mark it as Ready for Review

File: src/Nox.Types.EntityFramework/Types/Formula/FormulaConverter.cs [hunk 1]
```diff
@@ -0,0 +1,8 @@
+﻿using Microsoft.EntityFrameworkCore.Storage.ValueConversion;
+
+namespace Nox.Types.EntityFramework.Types;
+
+public class FormulaConverter : ValueConverter<Formula, string>
+{
+    public FormulaConverter() : base(dt => dt.Value, dtValue => Formula.From(dtValue)) { }
```
[the remark this answers is not in the document]
Comment (emir-sehovic1) [reply src/Nox.Types.EntityFramework/Types/Formula/FormulaConverter.cs]: Fixed, thanks

File: tests/Nox.Types.Tests/EntityFrameworkTests/Configuration/CountryDbConfiguration.cs
Comment (rochar) [inline tests/Nox.Types.Tests/EntityFrameworkTests/Configuration/CountryDbConfiguration.cs:1]: you can remove the db tests, this type will not be persisted on the database, will impact only the generators, we will generate a property with the formula

and the future dbconfiguration is no need, I will remove the ticket

thks

File: src/Nox.Types.EntityFramework/Types/Formula/FormulaConverter.cs
Comment (rochar) [inline src/Nox.Types.EntityFramework/Types/Formula/FormulaConverter.cs:1]: please remove the converter

This type will not be persisted, and used only in the generatores level

**Review by rochar — CHANGES_REQUESTED** [review]
looks good, small comments

**Comment by emir-sehovic1** [comment]
> looks good, small comments

Thanks for feedback @rochar , I've fixed the comments

**Review by rochar — APPROVED** [review]

**Pull request merged**